django/urls/resolvers.py

```diff
@@ -329,8 +334,22 @@ def check(self):
                     id="2_0.W001",
                 )
             )
+        warnings.extend(self._check_angle_bracket_pattern())
         return warnings
 
+    def _check_angle_bracket_pattern(self):
```

Comment:
IMO, `_check_pattern_unmatched_angle_bracket()` would be more consistent:
```suggestion
    def _check_pattern_unmatched_angle_bracket(self):
```